scripts/docs_version_pragma_check.sh

```diff
@@ -32,13 +32,54 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+function versionGreater() {
+    v1=$1
+    v2=$2
+    ver1=( ${v1//./ } )
+    ver2=( ${v2//./ } )
+
+    if (( ${ver1[0]} > ${ver2[0]} )); then
+        return 0
+    elif (( ${ver1[0]} == ${ver2[0]} )) && (( ${ver1[1]} > ${ver2[1]} )); then
+        return 0
+    elif (( ${ver1[0]} == ${ver2[0]} )) && (( ${ver1[1]} == ${ver2[1]} )) && (( ${ver1[2]} > ${ver2[2]} )); then
+        return 0
+    fi
+    return 1
+}
+function versionEqual() {
+    v1=$1
+    v2=$2
+    ver1=( ${v1//./ } )
+    ver2=( ${v2//./ } )
+
+    if (( ${ver1[0]} == ${ver2[0]} )) && (( ${ver1[1]} == ${ver2[1]} )) && (( ${ver1[2]} == ${ver2[2]} )); then
+        return 0
+    fi
+    return 1
+}
+
 printTask "Verifying that all examples from the documentation have the correct version range..."
 SOLTMPDIR=$(mktemp -d)
 (
     set -e
     cd "$SOLTMPDIR"
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/ docs
 
+    wget -q https://ethereum.github.io/solc-bin/bin/list.txt >/dev/null
```

Comment:
Better to use `-O` - just in case there is already a file with that name?

Reply:
Or directly pipe it into the while loop.

Reply:
Also, can you use `grep -e 'soljson-v[0-9]\+.[0-9]\+.[0-9]\++commit' ` so that it only contains released versions?

Reply:
Or do you actually need this to contain unreleased versions?

Reply:
I need only released versions, so will update like suggested